normits_demand/models/forecasting/ntem_forecast.py

```diff
@@ -172,6 +175,7 @@ def output_filename(
         trip_origin: str,
         purpose: int,
         year: int,
+        scenario: str = "Core",
```

Comment:
Does it makes sense to default to "Core" here? Would it make more sense to force the user to pick a name?

I'm concerned that with all the new scenarios in NTEM data, this could lead to unexpected errors.

This addition is also not documented

Reply:
Agree don't know why I did this.